src/tests/system/tests/test_sss_override.py

```diff
@@ -78,17 +108,15 @@ def test_sss_overrides__overriding_username_and_posix_attributes(client: Client,
 
 
 @pytest.mark.importance("high")
-@pytest.mark.topology(KnownTopology.LDAP)
-@pytest.mark.topology(KnownTopology.AD)
-@pytest.mark.topology(KnownTopology.Samba)
-def test_sss_overrides__overriding_group_name_and_gid(client: Client, provider: GenericProvider):
+@pytest.mark.topology([KnownTopology.LDAP, KnownTopology.AD, KnownTopology.Samba])
+@pytest.mark.preferred_topology(KnownTopology.LDAP)
+def test_sss_override__group(client: Client, provider: GenericProvider):
     """
-    :title: Locally overriding the name and GID of a group
+    :title: Override group
```

Comment:
Please make surr to add "i" to the assert messages in the assert loop to make clear what failed.